packages/sku/template/src/App/NextSteps.tsx

```diff
@@ -74,9 +72,6 @@ export function NextSteps({ environment }: NextStepsProps) {
                     </TextLink>
                     , <TextLink href="https://jestjs.io/">Jest</TextLink>,{' '}
                     <TextLink href="https://eslint.org/">ESLint</TextLink>,{' '}
```

Comment:
I think there's a double comma here now that it gets combined with the `, and more.` line below.

Reply:
Nice catch